drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -949,32 +985,45 @@ cosock.spawn(function()
             or Discovery.disco_api_instances[msg_device.device_network_id]
 
         if not api_instance then
-          api_instance = HueApi.new_bridge_manager("https://" .. bridge_ip, msg_device:get_field(Fields.API_KEY))
+          api_instance = HueApi.new_bridge_manager(
+            "https://" .. bridge_ip,
+            msg_device:get_field(Fields.API_KEY),
+            utils.labeled_socket_builder((msg_device.label or msg_device.device_network_id or msg_device.id or "unknown bridge"))
+          )
           Discovery.disco_api_instances[msg_device.device_network_id] = api_instance
         end
 
         found_bridges[msg_device.id] = msg.device
         local bridge_device_uuid = msg_device.id
 
-        log.info_with({ hub_logs = true }, "Querying bridge for devices from stray light handler")
+        -- TODO: We can optimize around this by keeping track of whether or not this bridge
```

Comment:
I know this was mentioned at some point, but for my own understanding - is this where we would implement a backoff for `process_strays` if we decide to do that in the future?

Reply:
Not necessarily; I was thinking we'd most likely do that check a bit further up in the `err == "timeout"` branch that starts on line 964, since that's the only place where we'd be scanning repeatedly in the event that everything else onboards properly.